.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018-2020 The OpenEBS Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: build
+
+on: ['push']
```

Comment:
Should this run on all push events or push events only on master?

Reply:
It should be for the master and release branches. (currently we have only master branch)
The reasoning is: This workflow produces the `ci` builds. So we need to make sure that if some patch comes to a release branch `branch-ci` tagged image would be available

Reply:
If that is the case should you add the branches to be triggered on this workflow as we've done for the others? Just to avoid being triggered on non-master and release branches?

Reply:
@xUnholy [This](https://github.com/openebs/node-disk-manager/blob/master/.github/workflows/build.yml#L16) is workflow that I referred. In the repo we won't be creating any other branches. Only master and release branches. If that is the case I think, push will be fine. Do you have any other corner cases that you see?

If even in the worst case we create another branch, it will be tagged  `:my-branch-ci`